src/vconsole/vconsole-setup.c

```diff
@@ -168,8 +168,9 @@ static int keyboard_load_and_wait(const char *vc, const char *map, const char *m
 }
 
 static int font_load_and_wait(const char *vc, const char *font, const char *map, const char *unimap) {
+        _cleanup_strv_free_ char **font_names = strv_split(font, " ");
```

Comment:
strv_split() may cause an oom failure. Unlikely, but we require all code to be oom safe. Please move this assignment below and check the return value. Unfortunately that'll make the initialization of `args` a bit harder. You may want to add a separate block `{   ...   }` to initialize `args` in.

Reply:
also, don't mix variable declarations and function invocations please in one line, see CODING_STYLE.